mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -141,6 +194,11 @@ export const WayPage = (props: WayPageProps) => {
             onChangeFinish={(text) => changeWayName(way, text)}
             isEditable={isOwner}
           />
+          <div>
+            {likes === 0 && "No likes"}
+            {likes === ONE_LIKE && `${likes} like`}
+            {typeof likes === "number" && likes > ONE_LIKE && `${likes} likes`}
```

Comment:
1. Likes could not be undefined semantically, type of likes should be always number
2. This logic looks like switch lite are cohesive but it's hard to understand if by code, let's make it simpler and just show line kinda: amount of users who add it to favorite: {length},

Reply:
Done.